solr/core/src/java/org/apache/solr/update/SolrIndexConfig.java

```diff
@@ -68,6 +68,19 @@
 
   public final double ramBufferSizeMB;
   public final int ramPerThreadHardLimitMB;
+  /**
```

Comment:
I appreciate documentation but _redundant_ documentation -- not so much.  Can't you do a @see or @link to one place -- `org.apache.lucene.index.IndexWriterConfig#setMaxCommitMergeWaitMillis`

Reply:
I didn't want to drop the link/see by itself, so I added a single line introducing the functionality. After that I added specifics about Solr (solrconfig configuration and the warning setting). I honestly don't see the problem here.